### ISSUE TITLE
functionality from sklearn cross_validation module moved to model_selection in 0.18

### ISSUE THREAD

**Issue by lefant** [paulgb/sklearn-pandas#68]
hi,

importing sklearn_pandas i get the following warning:

```
/usr/local/lib/python3.5/dist-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module 
into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will 
be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
```

this is because they moved some functions from cross_validation to the new model_selection module. read more here:
http://scikit-learn.org/stable/whats_new.html#version-0-18

best regards,
Fabian

**Comment by dukebody** [comment]
Thanks! I will update the code to try to import from the new module.

If you can do this yourself and submit a PR, it would be much welcomed. :)  Keep in mind we need to keep compatibility with sklearn versions less than 0.18

**Comment by dukebody** [comment]
@lefant See the PR. Looks good?

**Comment by dukebody** [comment]
Closed via #71.
